docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -0,0 +1,93 @@
+
+[float]
+[[find-events-analyze]]
+=== Find events to analyze
+
+You can only visualize events triggered by hosts configured with the Elastic Endpoint Security Integration. In KQL, this translates to any event with the `agent.type` set to `endpoint`.
+
+To access events that can be visually analyzed:
+
+1. In Elastic Security, select **Hosts** > **Events**. A list of all your host's events appears in the bottom panel of the host's view.
+
+2. Create a KQL query that filters all `endpoint` detected events by entering `agent.type: endpoint and process.entity_id : *` into the KQL search bar, and then select **Update**.
++
+[role="screenshot"]
+image::images/kql-agent-type.png[]
++
+A list of all `endpoint` related events appears.
+
+3. Select the **Analyze Event** icon next to the event you want to analyze. 
++
+[role="screenshot"]
+image::images/analyze-event.png[]
+
+The visual **Analyzer** view appears.
+
+[role="screenshot"]
+image::images/analyze-event-view.png[]
+
+You can also analyze events from <<timelines-ui,Timelines>>
+
+
+[discrete]
+[[visual-analyzer-ui]]
+=== Visual event analyzer UI
+
+Inside the visual analyzer, each cube represents either a node, process, or alert. Click and drag in timeline view to see all process relationships. 
+
+To make the analyzer full screen, select the **Full Screen** icon above the left-panel. 
+
+[role="screenshot"]
+image::images/full-screen-analyzer.png[]
+
+The left panel contains a list of all processes related to the event, starting with the event chain's first process. **Analyzed Events** are highlighted by a light blue outline around the cube.
+
+[role="screenshot"]
+image::images/process-list.png[]
+
+In the graphical view, you can:
+
+- Zoom in and out of the graphical view using the slider to the right of the timeline.
+- Click and drag around the graphical view to more process relationships.
+- See the time passed between each node.
+- See all events related to each process. 
+
+[role="screenshot"]
+image::images/graphical-view.png[]
+
+
+[discrete]
+[[process-and-event-details]]
+=== Process and event details
+
+To see more details about each related process, select the process either in the left panel or the graphical view. The information on the process appears in the left panel, which typically includes but isn't limited to:
+
+- The number of events associated with the process.
+- The timestamp when the process was executed.
+- The file path of the process within the host.
+- The `process-pid`.
+- The user name and domain that ran the process.
+- Any other relevant process information. 
+
+[role="screenshot"]
+image::images/process-details.png[]
+
+See event details by selecting the **# events** URL at the top of the process details view or choosing one of the event pills in the graphical view.
+
+
+Events are categorized based on their `event.type`. When selecting an `event.type`, a list of all those types of events appears in the left panel.
```

Comment:
I believe they are actually categorized by their `event.category`. @michaelolo24 @jonathan-buttner @kqualters-elastic Could one of your confirm?

Reply:
That's correct, it's by their `event.category`. Not sure if it's important to note this but technically the `event.category` can be an array, I think we just pick the first entry in the array and use that.

Reply:
Used `event.type` since it appears as `event.type` in the analyzer. I added a description that takes the `event-type` label in the UI into account.

Reply:
This seems like an issue. @jonathan-buttner @michaelolo24 @kqualters-elastic Would you find the label that @DonNateR is referring to an open an issue for 7.11 to fix it?